web/client/product/plugins.js

```diff
@@ -6,138 +6,143 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import {toModulePlugin} from "../utils/ModulePluginsUtils";
+
```

Comment:
Please replace require with import for static plugins

Reply:
Replaced